node/src/browserstack_session_capabilities.ts

```diff
@@ -1,7 +1,7 @@
 import { BstackOptions } from './bstack_options'
 
 export interface BrowserStackSessionCapabilities {
-  'bstack:options': BstackOptions
+  'bstack:options': BstackOptions & Record<string, unknown>
```

Comment:
Why not to add the missing fields to `BstackOptions` itself, instead of making the field type almost unrestrained?

If the type here should really be different from `BstackOptions`, please create a sub-type with the new fields.

Reply:
There's a schema on their side that controls it better and has way more options than what we want to add, this does give a bit more flexibility and not strictly extend their validation

Reply:
But AFAIK BrowserStack fails strictly when a field doesn't match the expected type. If the schema has too many fields, it's better to add only necessary fields and leave a comment telling that there are many other, and ideally containing a link to the schema.

Reply:
while not exactly being the schema, the set of capabilities are available [here](https://www.browserstack.com/automate/capabilities), however they aren't ideally presented. Maybe I could reach out to BrowserStack

Reply:
What we have is good enough